NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -530,13 +511,14 @@ class NeuroID
                     SessionStartResult(true, userID ?: "")
                 }
 
+            linkedSiteID = siteID
+            nidSamplingService.updateIsSampledStatus(linkedSiteID)
+            logger.d(msg="startSession() isSessionFlowSampled ${nidSamplingService.isSessionFlowSampled()} ${Companion.siteID}")
+
```

Comment:
The sampling status should be updated in 3 places, when start is called, when startSession is called, or LN 509-511 where we are just creating a linkedSiteID session. It should happen there because even if the user just calls start/startSession we might have throttling in place

Reply:
fixed. thanks!